app/src/main/java/net/measurementlab/ndt7/android/MainActivity.kt

```diff
@@ -1,32 +1,85 @@
 package net.measurementlab.ndt7.android
 
+import android.app.Activity
 import android.os.Handler
 import android.os.Looper
-import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
 import android.util.Log
+import android.widget.TextView
 import android.widget.Toast
+import butterknife.BindView
+import butterknife.ButterKnife
+import java.net.URL
+import com.google.gson.Gson
+import net.measurementlab.ndt7.android.model.Server
+import java.util.*
+import kotlin.math.pow
+
 
 private const val TAG = "MainActivity"
+private const val LOCATESERVER = "https://locate.measurementlab.net/ndt7"
+
+class MainActivity : Activity() {
+    @BindView(R.id.server)
+    lateinit var tv_server: TextView
```

Comment:
What's `lateinit`?

Reply:
Since I'm not an Kotlin expert, I added it because of Butterknife.